test/api/unit/services/BuildResolver.test.js

```diff
@@ -96,51 +96,43 @@ describe('BuildResolver', () => {
   describe('checking for branch on github', () => {
     const branch = 'staging';
     const sha = '123abc';
-    const user = factory.user();
-    const site = factory.site({ users: Promise.all([user]) });
-    let pValues;
 
-    beforeEach(() => {
-      pValues = Promise.props({
-        user,
-        site,
-      });
-    });
+    it('calls out to github if the branch cannot be found locally', async () => {
+      const user = await factory.user();
+      const site = await factory.site({ users: Promise.resolve([user]) });
```

Comment:
should this be Promise.all([user]) ?

Reply:
Not in this case because we already `await`ed the promise in the line above.

Reply:
This expects the value of the `users`  key to be a promise that resolves to an array of users, both `Promise.all([userPromise])` and `Promise.resolve([user])` do this..